src/handlers/transformation/transform-csv-to-json.ts

```diff
@@ -0,0 +1,15 @@
+import { S3Event } from "aws-lambda";
+import * as AWS from "aws-sdk";
+import csv from "csvtojson";
+
+// Unsure how to test this function or indeed if it even needs testing given that it basically just using S3 and the csvtojson library. Feedback/Help welcome!
+export async function transformCsvToJson(event: S3Event): Promise<any[]> {
+  const S3 = new AWS.S3();
+  const params = {
+    Bucket: event.Records[0].s3.bucket.name,
```

Comment:
This is only dealing with the first record. Can we be sure that we only get one? Or will we miss a file when two are dropped in at the same time?

Reply:
I've dropped two in at the same time and both were handled as expected. Each object int he S3 bucket transform the lambda function so it is working as expected.

Reply:
Added condition to check length and comments.